src/jobtypes.py

```diff
@@ -3,8 +3,8 @@
 import logging
 from tools.update_runnablejobs import update_runnableapi
 LOG = logging.getLogger(__name__)
-# JSONPATH = os.getcwd() + '/runnablejobs.json'
-JSONPATH = '/home/ubuntu/ouija/data/runnablejobs.json'
+JSONPATH = os.getcwd() + '/runnablejobs.json'
+# JSONPATH = '/home/ubuntu/ouija/data/runnablejobs.json'
```

Comment:
oh sorry, I just need to debug on local environment and forget to change it back. Sorry :(